src/sciwrappers.c

```diff
@@ -856,6 +856,18 @@ gint sci_find_text(ScintillaObject *sci, gint flags, struct Sci_TextToFind *ttf)
 	return (gint) SSM(sci, SCI_FINDTEXT, (uptr_t) flags, (sptr_t) ttf);
 }
 
+/** Sets the font for a particular style.
+ * @param sci Scintilla widget.
+ * @param style The style.
+ * @param font The font name.
+ * @param size The font (fractional) size. */
+void sci_set_font_fract(ScintillaObject *sci, gint style, const gchar *font, double size)
```

Comment:
we use GLib's type aliases, so `gdouble` (though it doesn't change anything here, that's only style)